src/models/TestCase.d.ts

```diff
@@ -9,6 +9,8 @@ declare class TestCase {
   errors: number;
   skipped: number;
   duration: number;
+  started: Date | undefined;
+  completed: Date | undefined;
```

Comment:
I recommend renaming them to `startTime` and `endTime` for slightly better clarity, as it aligns with conventions for timestamp properties.

Reply:
Ok, pushed changes for this.